<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_16" rowstretch="2,0" columnstretch="4,3,0">
    <item row="0" column="1" rowspan="2" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>Log</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4" rowstretch="1" columnstretch="0">
         <property name="topMargin">
          <number>25</number>
         </property>
         <item row="0" column="0">
          <widget class="QListWidget" name="lw_log"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_18">
      <item row="0" column="3">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Device 4</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_25" rowstretch="1,1" columnstretch="1,0,0,0,0">
         <item row="1" column="0" colspan="5">
          <layout class="QGridLayout" name="gridLayout_11">
           <property name="verticalSpacing">
            <number>20</number>
           </property>
           <item row="1" column="0">
            <widget class="QPushButton" name="dev4_stop">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="dev4_start">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" colspan="5">
          <layout class="QGridLayout" name="gridLayout_12" rowstretch="1,1,0" columnstretch="1,1">
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Term(sec)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="dev4_type">
             <item>
              <property name="text">
               <string>Normal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>H2S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NH3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CO</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cb_term_5">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Gas Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>PPM</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="line_ppm4"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Device 2</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_22" rowstretch="1,1" columnstretch="1,0,0,0,0">
         <item row="0" column="0" colspan="5">
          <layout class="QGridLayout" name="gridLayout_5" rowstretch="1,0,0" columnstretch="1,0">
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Gas Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cb_term_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Term(sec)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="dev2_type">
             <item>
              <property name="text">
               <string>Normal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>H2S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NH3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CO</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>PPM</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="line_ppm2"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="5">
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="verticalSpacing">
            <number>20</number>
           </property>
           <item row="1" column="0">
            <widget class="QPushButton" name="dev2_stop">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="dev2_start">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Device 1</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_21" rowstretch="1,1" columnstretch="1,0,0,0,0">
         <item row="0" column="0" colspan="5">
          <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0" columnstretch="1,0">
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="2" column="1">
            <widget class="QComboBox" name="cb_term">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="dev1_type">
             <item>
              <property name="text">
               <string>Normal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>H2S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NH3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CO</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="line_ppm"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Gas Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Term(sec)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>PPM</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="5">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="verticalSpacing">
            <number>20</number>
           </property>
           <item row="1" column="0">
            <widget class="QPushButton" name="dev1_stop">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="dev1_start">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Device 3</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_23" rowstretch="1,1" columnstretch="1,0,0,0,0">
         <item row="1" column="0" colspan="5">
          <layout class="QGridLayout" name="gridLayout_8">
           <property name="verticalSpacing">
            <number>20</number>
           </property>
           <item row="1" column="0">
            <widget class="QPushButton" name="dev3_stop">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="dev3_start">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" colspan="5">
          <layout class="QGridLayout" name="gridLayout_7" rowstretch="1,1,0" columnstretch="1,1">
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Term(sec)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cb_term_3">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="dev3_type">
             <item>
              <property name="text">
               <string>Normal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>H2S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NH3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CO</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Gas Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>PPM</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="line_ppm3"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Device 5</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_26" rowstretch="1,1" columnstretch="1,0,0,0,0">
         <item row="1" column="0" colspan="5">
          <layout class="QGridLayout" name="gridLayout_13">
           <property name="verticalSpacing">
            <number>20</number>
           </property>
           <item row="1" column="0">
            <widget class="QPushButton" name="dev5_stop">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="dev5_start">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" colspan="5">
          <layout class="QGridLayout" name="gridLayout_14" rowstretch="1,1,0" columnstretch="1,1">
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Term(sec)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="dev5_type">
             <item>
              <property name="text">
               <string>Normal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>H2S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NH3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CO</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Gas Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cb_term_6">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>PPM</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="line_ppm5"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
